code/Player/Clothes.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Sandbox;
+
+namespace ssl.Player
+{
+    public class Clothes : ModelEntity
+    {
+        public Clothes()
+        {
+        }
+
+        public Clothes(string clothesModel)
+        {
+            EnableShadowInFirstPerson = true;
+            EnableHideInFirstPerson = true;
+            AddCollisionLayer(CollisionLayer.Debris);
+            SetModel(clothesModel);
```

Comment:
fonction à part, comme pour moi ?

Reply:
Tu vas m'insulter mais ca donne ca
```suggestion
            public class Clothes : ModelEntity
    {
        public Clothes() { }

        public Clothes(string clothesModel)
        {
            EnableShadowInFirstPerson = true;
            EnableHideInFirstPerson = true;
            SetupModel(clothesModel);
        }

        private void SetupModel(string clothesModel)
        {
            AddCollisionLayer(CollisionLayer.Debris);
            SetModel(clothesModel);
        }
        
    }
```

Reply:
```cs
public class Clothes : ModelEntity
    {
        public Clothes() { }

        public Clothes(string clothesModel)
        {
            EnableShadowInFirstPerson = true;
            EnableHideInFirstPerson = true;
            SetupModel(clothesModel);
        }

        private void SetupModel(string clothesModel)
        {
            AddCollisionLayer(CollisionLayer.Debris);
            SetModel(clothesModel);
        }
        
    }
```
Mhhhh
En fait ne pas externaliser peut aussi etre bien 😥idk

Reply:
Bah moi ça m'irait de foutre directement dans le constructeur si le truc n'a pas à être appelé plus tard

Reply:
Faisons comme ca